model/schema.json

```diff
@@ -647,99 +645,152 @@
+          "type": "string",
+          "format": "uuid"
+        },
+        "@timestamp": {
+          "description": "Date/time the checkin was created",
+          "type": "string",
+          "format": "date-time"
+        },
+        "agent": { "$ref": "#/definitions/agent-metadata" },
+        "host": { "$ref": "#/definitions/host-metadata" },
+        "server": { "$ref": "#/definitions/server-metadata" },
+        "status": {
+          "description": "The current overall status of the Elastic Agent",
+          "type": "string",
+          "enum": ["enrolling", "healthy", "error", "degraded", "offline", "unenrolling", "upgrading"]
+        },
+        "message": {
+          "description": "The current overall status message of the Elastic Agent",
+          "type": "string"
+        },
+        "policy": {
+          "title": "Checkin Policy",
+          "description": "The current status of the applied policy",
+          "type": "object",
+          "properties": {
+            "id": {
+              "description": "The ID for the policy",
+              "type": "string",
+              "format": "uuid"
+            },
+            "revision": {
+              "description": "The revision of the policy",
+              "type": "integer"
+            },
+            "inputs": {
+              "title": "Checkin Policy Input",
+              "description": "The current input status per policy",
+              "type": "array",
+              "items": {
+                "type": "object",
+                "properties": {
+                  "id": {
+                    "description": "The ID for the input",
+                    "type": "string",
+                    "format": "uuid"
+                  },
+                  "template_id": {
+                    "description": "The template ID for the input",
+                    "type": "string",
+                    "format": "uuid"
+                  },
+                  "status": {
+                    "description": "The current status of the input",
+                    "type": "string",
+                    "enum": ["healthy", "error", "degraded"]
+                  },
+                  "message": {
+                    "description": "The current status message of the intput",
+                    "type": "string"
+                  }
+                },
+                "required": [
+                  "id",
+                  "template_id",
+                  "status",
+                  "message"
+                ]
+              }
+            }
+          },
+          "required": [
+            "id",
+            "revision",
+            "inputs"
+          ]
+        }
       },
-      "policy": {
-        "title": "Checkin Policy",
-        "description": "The current status of the applied policy",
+      "required": [
+        "_id",
+        "agent",
+        "status",
+        "message",
+        "enrolled_at",
+        "updated_at"
+      ]
+    },
+
+    "policy-data": {
```

Comment:
So far I've only added the top-level keys for the policy here, should we get more detailed?